devops/azure-pipelines/discord-job-template.yml

```diff
@@ -0,0 +1,25 @@
+jobs:
+  - job: DiscordMessage
+    dependsOn: ${{ parameters.dependOn }}
+    variables: 
+    - group: Discord-Webhook-Info
+    - name: discord_id
+      value: $[variables.discordid]
+    - name: discord_key
+      value: $[variables.discordkey]
```

Comment:
are you sure this is the right way to do this? i don't think it is. why are we making a second pair of variables when we already have the first pair

Reply:
There was an issue with variable  scope in Azure Pipelines that was confusing, but I found a solution to it, and we got rid of the doubled variables